src/array.cr

```diff
@@ -946,6 +946,17 @@ class Array(T)
     self
   end
 
+  # Optimized version of `Enumerable#skip`.
```

Comment:
Shouldn't this better repeat the description of `Enumerable#skip`?

Reply:
Yes. Actually, we should have some thingy to inherit doc from the parent. Let's work on that next (there are already a few methods in `Array` that say they are an optimized version of something else)

Reply:
For now I think it's better indeed, but it would be nice to have a `:previous_doc:` and/or `:super_doc:` to automatically copy the doc from previous def or super def or that same method

Reply:
I just copied the doc.